components/overflow-group/test/overflow-group.visual-diff.js

```diff
@@ -69,6 +69,9 @@ describe('d2l-overflow-group', () => {
 	const iconType = [
 		{
 			name: 'opener-type-mini-menu',
+			action: async() => {
+				await page.mouse.move(0, 0);
```

Comment:
This was a weird one... the mouse was randomly hovering over the third button for this test, I think because we click the third button in another test and then scroll to this one.  This fixes the problem, but I wasn't sure if there's a better way.

Reply:
Crazy! Could put a helper in the core visual-diff library like our resetFocus one, but probably worth waiting to see if the pops up in more places.